.github/workflows/check-code.yml

```diff
@@ -361,3 +361,29 @@ jobs:
             -w /workspace \
             paritytech/ci-unified:bullseye-1.74.0 \
             bash -c "bash -x scripts/run_benches_for_runtime.sh ${{ matrix.runtime }} dev"
+
+  trigger-gitlab:
+    name: Trigger GitLab CI
+    runs-on: ubuntu-latest
+    env:
+      # Change this in case of file renamings
```

Comment:
I would put this comment right next to the file name

Reply:
I can't. It's a script and I can't add comments to the script.

Reply:
Or you mean next line just next to it?

Reply:
just next to it      ` TARGET_WORKFLOW_FILE: trigger-gitlab-pipeline.yml  # Adjust the file name as needed`

Reply:
https://github.com/KILTprotocol/kilt-node/pull/755/commits/5a6b9980b8feac5d00ecbb488757b5eccbad5129